docs/advanced_usage.rst

```diff
@@ -23,6 +23,158 @@ You can also use a Unix Domain Socket to connect to the agent::
     tracer.configure(uds_path="/path/to/socket")
 
 
+Context
+-------
+
+The :class:`Context` object is used to represent the state of a trace at a
+point in time. It is used to propagate the trace across execution boundaries
+like processes (:ref:`Distributed Tracing <disttracing>`), threads and tasks.
+
+To retrieve the context of the currently active trace use::
+
+        context = tracer.current_trace_context()
+
+Note that if there is no active trace then ``None`` will be returned.
+
+
+Tracing Context Management
+--------------------------
+
+In ``ddtrace`` "context management" is the management of which
+:class:`ddtrace.Span` or :class:`ddtrace.context.Context` is active in an
+execution (thread, task, etc). There can only be one active span or context
+per execution.
```

Comment:
When do you have a context as opposed to an active span? Could be helpful to clarify why both are essentially treated as the same for the purposes of context management.

Reply:
Yeah contexts are used only for propagation (going from one thread to another, distributed tracing, etc). Good point that we should state when one would be active vs the other :+1:

Reply:
I've cleaned up the context section a bit more. I think it's clear now that both are interchangeable and that context is used to propagate a trace across execution bounds.